docs(fix): wording MTA-1561 (#202)
scaleway/docs-content: tutorials/setup-mysql/index.mdx

- You have an account and are logged into the [Scaleway Console](https://console.scaleway.com)
- You have [configured your SSH Key](/console/my-project/how-to/create-ssh-key/)

> - You have a Virtual Instance running on Ubuntu Focal Fossa (20.04)

Reviewer: ```suggestion
- You have an Instance running on Ubuntu Focal Fossa (20.04)
```
